### PR TITLE
NEST-407: Create media theme deployment packages automatically

### PR DESCRIPTION
**Pull request by wAsnk**
[NEST-407](https://lombiq.atlassian.net/browse/NEST-407)

[NEST-407]: https://lombiq.atlassian.net/browse/NEST-407?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Lombiq.Hosting.MediaTheme.Targets/Lombiq.Hosting.MediaTheme.Targets.csproj [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <IsPublishable>false</IsPublishable>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Title>Lombiq Hosting - Media Theme Targets</Title>
+    <Authors>Lombiq Technologies</Authors>
+    <Copyright>Copyright © 2023, Lombiq Technologies Ltd.</Copyright>
+    <Description>Lombiq Hosting - Media Theme Targets: Provides automatic media theme recipe and tempalte updating. See the project website for detailed documentation.</Description>
+    <PackageIcon>NuGetIcon.png</PackageIcon>
+    <PackageTags>Lombiq;MediaTheme;MSBuild</PackageTags>
```
Comment (DemeSzabolcs) [inline Lombiq.Hosting.MediaTheme.Targets/Lombiq.Hosting.MediaTheme.Targets.csproj]: I copied these from the Deployer's csproj to make these consistent.

```suggestion
    <PackageTags>OrchardCore;Lombiq;AspNetCore;MSBuild,Media;Targets</PackageTags>
```

File: Lombiq.Hosting.MediaTheme.Targets/.config/dotnet-tools.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "lombiq.hosting.mediatheme.deployer": {
+      "version": "2.1.2-alpha.0.nest-407-5",
```
Comment (DemeSzabolcs) [inline Lombiq.Hosting.MediaTheme.Targets/.config/dotnet-tools.json]:
```suggestion
      "version": "3.0.0-alpha.0.nest-407",
```

**Review by DemeSzabolcs — CHANGES_REQUESTED** [review]

File: Lombiq.Hosting.MediaTheme.Targets/Readme.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Lombiq Hosting - MediaTheme MSBuild Targets
+
+[![Lombiq.Hosting.MediaTheme.Targets NuGet](https://img.shields.io/nuget/v/Lombiq.Hosting.MediaTheme.Targets?label=Lombiq.Hosting.MediaTheme.Targets)](https://www.nuget.org/packages/Lombiq.Hosting.MediaTheme.Targets/)<!-- #spell-check-ignore-line -->
```
Comment (0liver) [inline Lombiq.Hosting.MediaTheme.Targets/Readme.md]: Why do have to ignore this line?

File: Lombiq.Hosting.MediaTheme.Targets/Readme.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## About
+
+Provides automatic media theme package creation in the project. This way you don't have to manage changes in your media theme recipe, or folder structure. The targets reuses the media-theme-deployer dotnet tool will be automatically installed locally to the project, then it will create the deployment zip file. Please read the [Lombiq.Hosting.MediaTheme Readme](https://github.com/Lombiq/Hosting-Media-Theme/blob/dev/Readme.md) for more information about this deployer tool.<!-- #spell-check-ignore-line -->
```
Comment (0liver) [inline Lombiq.Hosting.MediaTheme.Targets/Readme.md]: Why do have to ignore this line?

File: Lombiq.Hosting.MediaTheme.Targets/Readme.md (hunk 3 above)
Comment (wAsnk) [reply Lombiq.Hosting.MediaTheme.Targets/Readme.md]: Somehow there are errors for these two lines even if there are no double spaces, and it looks like the ignore didn't work as well. https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/4437936623?pr=377

File: Lombiq.Hosting.MediaTheme.Targets/Readme.md (hunk 4 above)
Comment (0liver) [reply Lombiq.Hosting.MediaTheme.Targets/Readme.md]: Here, the next line starts without an empty line in between. You need to put text either on the same line, or separate it by an empty line.

File: Lombiq.Hosting.MediaTheme.Targets/Readme.md (hunk 3 above)
Comment (0liver) [reply Lombiq.Hosting.MediaTheme.Targets/Readme.md]: The error is not from the spell checker but the Markdown linter in Node.js Extensions 😉

Comment (wAsnk) [reply Lombiq.Hosting.MediaTheme.Targets/Readme.md]: Okay, but what's the problem with this line?

Comment (0liver) [reply Lombiq.Hosting.MediaTheme.Targets/Readme.md]: I checked and I honestly don't know.

Comment (wAsnk) [reply Lombiq.Hosting.MediaTheme.Targets/Readme.md]: Now I added `<!-- markdownlint-disable -->` but that didn't help either... Also I couldn't find this "_doubled-spaces: Found doubled spaces._" error anywhere documented in markdownlint. :O

Comment (wAsnk) [reply Lombiq.Hosting.MediaTheme.Targets/Readme.md]: Oh it is actually textlint

File: Lombiq.Hosting.MediaTheme.Targets/.config/dotnet-tools.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "lombiq.hosting.mediatheme.deployer": {
+      "version": "3.0.0-alpha.1.nest-407",
```
Comment (DemeSzabolcs) [inline Lombiq.Hosting.MediaTheme.Targets/.config/dotnet-tools.json]:
```suggestion
      "version": "3.0.0-alpha.3.nest-407",
```

**Review by DemeSzabolcs — CHANGES_REQUESTED** [review]

**Review by DemeSzabolcs — APPROVED** [review]

**Pull request merged**